entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # Copyright (c) 2022 MobileCoin Inc.
 
+[[ "$INPUT_DEBUG" == 'true' ]] && set -x
```

Comment:
To be honest this scares me.  I know that GHA does a pretty good job of masking secrets output, but its just doing string matching and its not perfect. Setting `-x` will probably lead to secrets exposure.

I'd prefer we set specific debug lines for things we want to output:
```
[ -n "${INPUT_DEBUG}" ] && echo "## Create target dir ${target_dir}"
```

There's a "proper" way to set debug on an action: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging

I'm not exactly sure how that's presented to the github action.

Reply:
Or make a function like `debug_echo` to make it a bit more dry.

Reply:
Fair enough. I'll add `debug_echo`